packages/core/review-workflows/admin/src/services/content-manager.ts

```diff
@@ -16,112 +18,128 @@ interface ContentTypes {
+              id: slug !== SINGLE_TYPES ? `${model}_${id}` : model,
+            },
+            { type: 'Document', id: `${model}_LIST` },
+            'ReviewWorkflowStages',
+          ];
         },
       }),
-      transformResponse: (res: UpdateAssignee.Response) => res.data,
-      invalidatesTags: (_result, _error, { slug, id, model }) => {
-        return [
-          {
-            type: 'Document',
-            id: slug !== SINGLE_TYPES ? `${model}_${id}` : model,
+      updateAssignee: builder.mutation<
+        UpdateAssignee.Response['data'],
+        UpdateAssignee.Request['body'] & UpdateAssignee.Params & { slug: string; params?: object }
+      >({
+        query: ({ model, slug, id, params, ...data }) => ({
+          url: `/review-workflows/content-manager/${slug}/${model}/${id}/assignee`,
+          method: 'PUT',
+          data,
+          config: {
+            params,
           },
-          { type: 'Document', id: `${model}_LIST` },
-        ];
-      },
-    }),
-    getContentTypes: builder.query<ContentTypes, void>({
-      query: () => ({
-        url: `/content-manager/content-types`,
-        method: 'GET',
+        }),
+        transformResponse: (res: UpdateAssignee.Response) => res.data,
+        invalidatesTags: (_result, _error, { slug, id, model }) => {
+          return [
+            {
+              type: 'Document',
+              id: slug !== SINGLE_TYPES ? `${model}_${id}` : model,
+            },
+            { type: 'Document', id: `${model}_LIST` },
+            'RecentlyAssignedList',
+          ];
+        },
       }),
-      transformResponse: (res: { data: Contracts.ContentTypes.ContentType[] }) => {
-        return res.data.reduce<ContentTypes>(
-          (acc, curr) => {
-            if (curr.isDisplayed) {
-              acc[curr.kind].push(curr);
+      getContentTypes: builder.query<ContentTypes, void>({
+        query: () => ({
+          url: `/content-manager/content-types`,
+          method: 'GET',
+        }),
+        transformResponse: (res: { data: Contracts.ContentTypes.ContentType[] }) => {
+          return res.data.reduce<ContentTypes>(
+            (acc, curr) => {
+              if (curr.isDisplayed) {
+                acc[curr.kind].push(curr);
+              }
+              return acc;
+            },
+            {
+              collectionType: [],
+              singleType: [],
             }
-            return acc;
-          },
-          {
-            collectionType: [],
-            singleType: [],
-          }
-        );
-      },
+          );
+        },
+      }),
+      getRecentlyAssignedDocuments: builder.query<
+        Homepage.GetRecentlyAssignedDocuments.Response['data'],
+        void
+      >({
+        query: () => '/review-workflows/homepage/recently-assigned-documents',
+        transformResponse: (response: Homepage.GetRecentlyAssignedDocuments.Response) =>
+          response.data,
+        providesTags: (_, _err) => ['RecentlyAssignedList'],
```

Comment:
i think the list data would get stale when the content of an entry displayed in the widget gets updated. maybe you could also provide RecentDocumentList here?